tests/test_daemon_script.py

```diff
@@ -49,6 +50,27 @@ def test_loading_valid_conf(self):
             }
         )
 
+    def test_loading_valid_conf_with_password(self):
```

Comment:
Small remark about naming: what you did is not only for passwords, but for any configuration parameter containing an environment variable.

Reply:
Done in the next commit